easybuild/easyconfigs/c/CellBender/CellBender-0.2.2-foss-2022a.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'CellBender'
+version = '0.2.2'
+
+homepage = 'http://github.com/broadinstitute/CellBender'
+description = """
+CellBender is a software package for eliminating technical artifacts from
+high-throughput single-cell RNA sequencing (scRNA-seq) data.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('scikit-learn', '1.1.2'),
+    ('matplotlib', '3.5.2'),
+    ('anndata', '0.8.0'),
+    ('PyTables', '3.8.0'),
+    ('pyro-ppl', '1.8.4'),
+]
+
+exts_list = [
+    ('cellbender', version, {
+        'preinstallopts': "sed -i '/^sphinx/d' REQUIREMENTS.txt && ",
```

Comment:
@lexming could you add a comment about this please?

Reply:
sure, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/19296/commits/01e492f27e41fe8a03fa1d9972d869a68db111d3